recipies/llama-3-70b/model/model-download.yaml

```
@@ -0,0 +1,51 @@
+                  key: token
+          args:
+            - |
+              set -eux
+              pip install --no-cache-dir huggingface_hub hf_transfer
+              export HF_HUB_ENABLE_HF_TRANSFER=1
+              huggingface-cli download $MODEL_NAME
+          volumeMounts:
+            - name: model-cache
+              mountPath: /root/.cache/huggingface/hub
+      volumes:
+      - name: model-cache
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)